shared/common-adapters/input.native.js

```diff
@@ -4,7 +4,7 @@
 import React, {Component} from 'react'
 import Box from './box'
 import Text, {getStyle as getTextStyle} from './text'
-import {NativeTextInput} from './native-wrappers.native'
```

Comment:
You should change this in native wrappers so it's pulled in by everything. We already patch native components there

Reply:
👍